Dockerfile.win10.min

```diff
@@ -37,9 +37,9 @@ RUN choco install unzip -y
 #
 # Installing TensorRT
 #
-ARG TENSORRT_VERSION
-ARG TENSORRT_ZIP="TensorRT-${TENSORRT_VERSION}.Windows10.x86_64.cuda-12.0.zip"
-ARG TENSORRT_SOURCE=${TENSORRT_ZIP}
+ARG TENSORRT_VERSION=10.0.1.6
+ARG TENSORRT_ZIP="TensorRT-${TENSORRT_VERSION}.Windows10.x86_64.cuda-12.4.zip"
+ARG TENSORRT_SOURCE=https://developer.nvidia.com/downloads/compute/machine-learning/tensorrt/10.0.1/zip/TensorRT-10.0.1.6.Windows10.win10.cuda-12.4.zip
```

Comment:
Any reason for hard coding this rather than using the parameter defined above?

Reply:
Usually I don't update sources URLs. I can revert is for the next time. (in fact this value is ephemeral, so it more for customer convenience)